cdci_data_analysis/flask_app/app.py

```diff
@@ -247,10 +247,10 @@ def push_renku_branch():
     # TODO could not think of better name
     required_roles = ['renku contributor']
     if not all(item in roles for item in required_roles):
-        lacking_roles = ", ".join(sorted(list(set(required_roles) - set(roles))))
+        lacking_roles = "\n".join(['- <b>' + r + '</b>' for r in sorted(list(set(required_roles) - set(roles)))])
```

Comment:
dispatcher responses are also meant for python API, so it's not good that they have HTML.
Instead, frontend might try to render output with HTML. Replace \n with   <br>, etc.

Reply:
Taken care of in: [https://github.com/oda-hub/mmoda-frontend-module/pull/149](https://github.com/oda-hub/mmoda-frontend-module/pull/149)